app/components/Header.tsx

```diff
@@ -1,6 +1,9 @@
 "use client";
+
+// LIBRARY IMPORTS
 import { Link as ChakraLink } from "@chakra-ui/next-js";
 import { Box, Grid, Stack, IconButton, Spacer } from "@chakra-ui/react";
+import { FaBars } from "react-icons/fa6";
```

Comment:
replaced the imported image with an icon as icons have more built-in accessibility than images and it's more consistent